[AV-68634] Add docs for setting up clusters and data models (#318)
couchbaselabs/docs-devex: modules/tutorials/pages/create-couchbase-cluster.adoc

Reviewer: Can you also delete `.install-couchbase-server.adoc` if you have not done so already?

Author: Done.

= Create and Deploy a Cluster with Capella Free Tier
:description: Learn how to create and deploy a cluster using Capella Operational Free Tier.
:page-topic-type: tutorial
:page-pagination: full
:page-toclevels: 2
:navtitle: Create and Deploy a Cluster

[abstract]
{description}

== Prerequisites

> Before starting this tutorial, you must have a Couchbase Capella account. If you do not have one already, xref:cloud:get-started:create-account.adoc[Create an Account]. 

Reviewer 2: No more than one sentence per line.

Author: Done.

== Create and Deploy a Cluster

> Every Capella account includes one free tier operational cluster. In this tutorial, you will use the free tier cluster to create and manage student records. 

Reviewer 2: No more than one sentence per line.

Author: Done.

= Create and Deploy a Cluster with Capella Free Tier
:description: Learn how to create and deploy a cluster using Capella Operational Free Tier.
:page-topic-type: tutorial
:page-pagination: full
:page-toclevels: 2
:navtitle: Create and Deploy a Cluster

[abstract]
{description}

== Prerequisites

Before starting this tutorial, you must have a Couchbase Capella account. 
If you do not have one already, xref:cloud:get-started:create-account.adoc[Create an Account]. 

== Create and Deploy a Cluster

Every Capella account includes one free tier operational cluster. 
In this tutorial, you will use the free tier cluster to create and manage student records. 

To create and deploy a cluster: 

. Sign in to https://cloud.couchbase.com/[Couchbase Capella]. 

. On the *Operational Clusters* page, click btn:[Create Cluster].

> . Select *My First Project* as the project for your cluster. 

Reviewer: Need to allow for the case where they've already created a project for whatever reason

```suggestion
. Select *My First Project* as the project for your cluster.
(If you've already created a project, you can select that instead.)
```

. Under *Cluster Option*, select btn:[Free].

. In the *Cluster Name* field, enter *student-cluster*.

. (Optional) Provide a description of your cluster.

. Select one of the available cloud service providers.  

. Select an available geographic region for your cluster.
+

> TIP: If you are unsure which cloud provider and region to choose, select AWS and US East (N.Virginia).

Reviewer: This makes it look like the users need to change something whereas they actually can just accept the defaults. Also, while the default region was US East for me, it wasn't N.Virginia, so I guess the precise location may change. Maybe omit that bit.

```suggestion
TIP: If you are unsure which cloud provider and region to choose, select the default options, for example *AWS* and *US East*.
```

. Accept the default *CIDR Block* for your cluster. 

. Click btn:[Create Cluster] to deploy the cluster.

== Next Steps

> After deploying a cluster, you can xref:buckets-scopes-and-collections.adoc[implement a data model].

Reviewer: They won't be able to use the cluster right away.

```suggestion
The cluster may take a few minutes to deploy.
When the cluster is deployed, you can xref:buckets-scopes-and-collections.adoc[implement a data model].
```
